Framework/DataHandling/src/LoadEventNexus.cpp

```diff
@@ -210,27 +185,11 @@ void LoadEventNexus::init() {
                                                        Direction::Input),
                   "Load the monitors from the file (optional, default False).");
 
-  declareProperty(
-      make_unique<PropertyWithValue<bool>>("MonitorsAsEvents", false,
-                                           Direction::Input),
-      "If present, load the monitors as events. '''WARNING:''' WILL "
-      "SIGNIFICANTLY INCREASE MEMORY USAGE (optional, default False). ");
-
-  declareProperty("LoadEventMonitors", true,
-                  "Load event monitor in NeXus file both event monitor and "
-                  "histogram monitor found in NeXus file."
-                  "If both of LoadEventMonitor and LoadHistoMonitor are true, "
-                  "or both of them are false,"
-                  "then it is in the auto mode such that any existing monitor "
-                  "will be loaded.");
-
-  declareProperty("LoadHistoMonitors", true,
-                  "Load histogram monitor in NeXus file both event monitor and "
-                  "histogram monitor found in NeXus file."
-                  "If both of LoadEventMonitor and LoadHistoMonitor are true, "
-                  "or both of them are false,"
-                  "then it is in the auto mode such that any existing monitor "
-                  "will be loaded.");
+  std::vector<std::string> options{"", "Events", "Histogram"};
```

Comment:
So this change will break user scripts - should we provide a deprecation warning in the Python layer?

Reply:
In common usage, people use the default which is "figure it out." It is only in the case of both histogram *and* events being in the file that the user has to intervene.